CMakeLists.txt

```diff
@@ -123,6 +123,16 @@ if(NOT TARGET xxhash)
   add_subdirectory(lib/xxhash/cmake_unofficial EXCLUDE_FROM_ALL)
 endif()
 
+# Installation Directories
+# ========================
+if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+  set(CFL_INSTALL_BINDIR "bin")
+  set(CFL_INSTALL_LIBDIR "lib")
+else()
+  set(CFL_INSTALL_BINDIR ${CMAKE_INSTALL_FULL_BINDIR})
```

Comment:
I just followed what Fluent Bit did here but you may want another specific directory.